drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -208,7 +213,7 @@ def self.source_for(destination_id:, user:)
   end
 
   def ssn_serial
-    self.SSN&.[](-4..-1)
+    self.SSN&.[](-4..)
```

Comment:
Suggest reverting this unrelated change? Is there a bug?

Reply:
Opened https://github.com/greenriver/hmis-warehouse/pull/5267 to separate this out

Reply:
Gotcha thanks!